stewart/parts/header.html

```diff
@@ -0,0 +1,6 @@
+<!-- wp:group {"align":"full","layout":{"type":"flex","justifyContent":"space-between"},"style":{"spacing":{"padding":{"top":"var(--wp--custom--margin--horizontal, 30px)","bottom":"40px"}}},"className":"site-header"} -->
```

Comment:
I think we should remove this class

Reply:
then we are missing the responsive styles: https://github.com/Automattic/themes/blob/trunk/blockbase/sass/base/_header.scss

Reply:
This isn't a Blockbase theme

Reply:
sorry I didn't check!